workspace/src/common/launch/art_localization_launch/launch/art_localization.launch.py

```diff
@@ -1,113 +1,20 @@
-# OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.#
-
 # ros imports
 from launch import LaunchDescription
-from launch_ros.actions import Node
 
 # internal imports
-from launch_utils import AddLaunchArgument, GetLaunchArgument, GetPackageSharePath
-from enum import Enum
-from localization_py.state_estimation import EstimationAlgorithmOption
+from launch_utils import IncludeLaunchDescriptionWithCondition
 
 
 def generate_launch_description():
     ld = LaunchDescription()
 
-    # ----------------
-    # Launch Arguments
-    # ----------------
-
-    AddLaunchArgument(ld, "art_localization/input/gps", "/sensing/gps/data")
-    AddLaunchArgument(
-        ld, "art_localization/input/magnetometer", "/sensing/magnetometer/data"
-    )
-    AddLaunchArgument(ld, "art_localization/input/gyroscope", "/sensing/gyroscope/data")
-    AddLaunchArgument(
-        ld, "art_localization/input/accelerometer", "/sensing/accelerometer/data"
-    )
-    AddLaunchArgument(
-        ld, "art_localization/input/vehicle_inputs", "/control/vehicle_inputs"
-    )
-    AddLaunchArgument(
-        ld, "art_localization/output/filtered_state", "/vehicle/filtered_state"
-    )
-
-    AddLaunchArgument(
-        ld,
-        "estimation_alg",
-        EstimationAlgorithmOption.GROUND_TRUTH.value,
-        choices=[o.value for o in EstimationAlgorithmOption.__members__.values()],
-    )
+    # ---------------
+    # Launch Includes
+    # ---------------
 
-    # -----
-    # Nodes
-    # -----
-
-    node = Node(
-        package="localization_py",
-        executable="state_estimation",
-        name="state_estimation",
-        remappings=[
-            ("~/input/gps", GetLaunchArgument("art_localization/input/gps")),
-            (
-                "~/input/magnetometer",
-                GetLaunchArgument("art_localization/input/magnetometer"),
-            ),
-            (
-                "~/input/gyroscope",
-                GetLaunchArgument("art_localization/input/gyroscope"),
-            ),
-            (
-                "~/input/accelerometer",
-                GetLaunchArgument("art_localization/input/accelerometer"),
-            ),
-            (
-                "~/input/vehicle_inputs",
-                GetLaunchArgument("art_localization/input/vehicle_inputs"),
-            ),
-            (
-                "~/output/filtered_state",
-                GetLaunchArgument("art_localization/output/filtered_state"),
-            ),
-        ],
-        parameters=[
-            GetPackageSharePath(
-                "art_localization_launch", "config", "4DOF_dynamics.yml"
-            ),
-            GetPackageSharePath("art_localization_launch", "config", "EKF_param.yml"),
-            {"use_sim_time": GetLaunchArgument("use_sim_time")},
-            {"estimation_alg": GetLaunchArgument("estimation_alg")},
-        ],
+    # IncludeLaunchDescriptionWithCondition(ld, "art_localization_launch", "ekf_estimation")
```

Comment:
why is this commented out?

Reply:
What is the right way to only include this if the metapackage wants ekf? so only include the launch description for the localizaiton package that is getting built?

Reply:
Probs use the auto generationed `disable_ekf_estimation` flag and set it automatically in the launch script.